addons/base_iban/static/src/components/iban_widget/iban_widget.scss

```diff
@@ -4,7 +4,7 @@
     .o_iban {
         display: inline-flex;
         margin-left: -15px;
-        margin-top: 5px;
+        //margin-top: 5px;
```

Comment:
to delete

Reply:
remove unused classes too
`o_iban_input_with_validator` not used

Reply:
Finally, I kept it and just changed a bit the number of pixel.
It was solving an issue I didn't spot.